Dockerfile

```diff
@@ -6,11 +6,16 @@ COPY Gemfile.lock /usr/src/app/
 
 RUN apk --update add ruby ruby-dev ruby-bundler build-base && \
     bundle install -j 4 && \
-    apk del build-base && rm -fr /usr/share/ri
+    apk del build-base && rm -fr /usr/share/ri && \
+    apk --update add git
```

Comment:
Just separating it from the ruby specific packages. I can add it in to the above if that's preferable.